src/components/createAppointment/timePicker/timePicker.js

```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from 'react'
+import { PickerWrapper } from '../createAppointment.styles'
+import { Text } from '../createAppointment.styles'
+import { Listing, Button } from '../timePicker/timePicker.styles'
+
+const data = [
```

Comment:
Its hard to undertand what is data includes, probably you should rename it to understandable variable

Reply:
renamed